src/styles/components/_home-search-box.scss

```
@@ -0,0 +1,19 @@
+@import '../abstracts/_abstracts-dir';
+
+.home-search-box {
+  @include flex-center();
+
+  &__button {
+    @include button-hover($picton-blue);
+
+    background: $picton-blue;
```

Avoid using shorthand properties for only one value.
Ref: https://nimblehq.co/compass/development/code-conventions/css/
```suggestion
    background-color: $picton-blue;
```